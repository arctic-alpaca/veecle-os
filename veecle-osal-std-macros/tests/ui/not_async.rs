#[veecle_osal_std_macros::main]
fn main() {}
