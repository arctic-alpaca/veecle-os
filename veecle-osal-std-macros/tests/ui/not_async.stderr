error: function must be `async`
 --> tests/ui/not_async.rs:2:1
  |
2 | fn main() {}
  | ^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/not_async.rs:2:13
  |
2 | fn main() {}
  |             ^ consider adding a `main` function to `$DIR/tests/ui/not_async.rs`
